File: lang/python/test_sprite.py
import cog

cog.init()

test_sprite_id = cog.sprite_add("media/test0.png")
test = cog.sprite()
test.dim = cog.dim2()
test.dim.w = 1.0
test.dim.h = 1.0
cog.sprite_set(test_sprite_id, test)

while not cog.hasquit():
    cog.loopstep()
cog.quit()

File: lang/lua/luajit/test_sprite.lua
local cog = require 'cog'
local ffi = require("ffi")

cog.init()
local COG_PI = 3.14

local test_sprite_id = cog.sprite.add("media/test0.png")
local test = cog.sprite.new()
test.dim = cog.dim2.new(1.0, 1.0)
cog.sprite.set(test_sprite_id, test)

while not cog.hasquit() do
    cog.loopstep()
end
cog.quit()
